apps/dashboard/app/assets/stylesheets/apps.scss

```diff
@@ -38,6 +38,20 @@ tr.app {
    font-size: 14px;
 }
 
+.apps-table .app-icon {
+  width: 24px;
+  height: 24px;
+  font-size: 14px;
+}
+
+.apps-table p {
+  display: inline-block;
+}
+
+.apps-table ul {
+  min-width: 210px;
+}
```

Comment:
Not sure how to get the list items to be flexible. 

Without this the box is not as wide and some paths are shown on a second line from their icon.
![image](https://user-images.githubusercontent.com/4874123/107794754-d33add80-6d25-11eb-8bd0-bc3c06ccef6e.png)